README.md

```diff
@@ -828,31 +817,30 @@ example the `drone` secret is exposed as `/run/secrets/drone-server` for
 
 ## Migrate from pass/krops
 
-If you have used [pass](https://www.passwordstore.org) before i.e. in
-[krops](https://github.com/krebs/krops) than you can use the following one-liner
-to convert all your secrets to a yaml structure.
+If you have used [pass](https://www.passwordstore.org) before (e.g. in
+[krops](https://github.com/krebs/krops)) than you can use the following one-liner
+to convert all your secrets to a YAML structure:
 
 ```console
 $ for i in *.gpg; do echo "$(basename $i .gpg): |\n$(pass $(dirname $i)/$(basename $i .gpg)| sed 's/^/  /')"; done
```

Comment:
This one-liner doesn't work; it doesn't handle nested directories, which show up commonly if one is using `pass`. To be honest, I'm not entirely sure it's easy to come up with a one-liner for this because of how convoluted the YAML format is (how do we handle special characters e.g. `\0` or `\r`?) The easiest approach I can think of is somehow encoding to JSON, then using `yq -P` to translate to idiomatic YAML.

Reply:
This was only used for krops at a time where there was no support for nesting anyway.

Reply:
We could also drop this otherwise. I don't think there are too many krops user, that not migrated yet.

Reply:
Let's keep this for another PR.